README.md

```diff
@@ -162,9 +162,13 @@ sqlx = { version = "0.4.0-beta.1", default-features = false, features = [ "runti
 
 ### Quickstart
 
-```rust
-use std::env;
+```toml
+[dependencies]
+sqlx = { version = "0.4.0-beta.1", features = [ "postgres" ] }
+async-std = { version = "1.6", features = [ "attributes" ] }
```

Comment:
Could you add a commented out line for `tokio` to match the commented out attribute for `#[tokio::main]`
Ideally there'd be some kind of thing to match the
```rs
// use sqlx::mysql::MySqlPoolOptions;
// etc.
```
lines too, but I don't find that as important